go.mod

```diff
@@ -38,8 +38,8 @@ require (
 	k8s.io/apiserver v0.17.6
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
 	k8s.io/utils v0.0.0-20200603063816-c1c6865ac451
-	knative.dev/pkg v0.0.0-20200810090030-095ff27da65c
-	knative.dev/test-infra v0.0.0-20200806191129-68b7defbd189
+	knative.dev/pkg v0.0.0-20200811165506-f6ed1766e8ee
```

Comment:
this is the same issue we saw before in knative-gcp. It is not being set to v0.17 although you upgraded to that

Reply:
The git hash (f6ed1766e8ee) is the correct version. I think it might be that go.mod will record tags, but not branches. So we don't see anything but an empty version with a timestamp.